docs/src/main/asciidoc/_galleon/Galleon_layers.adoc

```diff
@@ -452,7 +452,7 @@ link:#gal.io[io] +
 link:#gal.base-server[base-server] +
 
 |[[gal.resource-adapters]]resource-adapters
-|Support for deployment of JCA adapters.
+|Support for deployment of Jakarta Connectors adapters.
```

Comment:
It was wrong before calling it just "adapters" instead of "resource adapters". I think it's the opportunity to fix it as "Jakarta Connectors resource adapters"